types/stellar-sdk/tslint.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Please don't change or disable the lint rules.

Reply:
Fixed